docs/USAGE.md

```diff
@@ -3,26 +3,43 @@
 The Earthfile configuration in this directory facilitates builds of Space ROS from source code.
 The generated container image is based on Ubuntu 22.04 (Jammy)
 
-## Building the Docker Image
+## Prerequisites
 
-The [Earthly](https://earthly.dev/get-earthly) utility is required to build this image.
+The following software is required to build the Space ROS Docker image:
 
-To build the image, run:
+- [Docker](https://docs.docker.com/get-docker/)
+- [Earthly](https://earthly.dev/get-earthly)
+- [Git](https://git-scm.com/downloads)
+
+## Setup
+
+The image is built using the [Earthly](https://earthly.dev/get-earthly) utility. First, clone the Space ROS repository:
 
 ```bash
-./build.sh
+git clone https://github.com/space-ros/space-ros.git
+cd space-ros
 ```
+Make sure, docker is running and you have the necessary permissions to run docker commands.
 
-The build process will take about 20 or 30 minutes, depending on the host computer.
+SpaceROS comes in different flavors, the following image variants are available:
```

Comment:
I think we use a space?

```suggestion
Space ROS comes in different flavors, the following image variants are available:
```

Reply:
I will take this suggestion to overall changes in the documentation.